data_safe_haven/infrastructure/stacks/sre/remote_desktop.py

```diff
@@ -295,42 +307,40 @@ def __init__(
                     ),
                 ),
                 containerinstance.ContainerArgs(
-                    image="ghcr.io/alan-turing-institute/guacamole-user-sync:v0.2.0",
+                    image="ghcr.io/alan-turing-institute/guacamole-user-sync:v0.4.0",
```

Comment:
I don't suppose Apricot could help not need to do this?

Could we translate Entra groups into the groups Guacamole looks for?

Reply:
Potentially we could do this in future - I'd prefer to keep Apricot agnostic about what its LDAP tree looks like though, and not tie it too closely to the Guacamole structure.

Reply:
Not changing here.